content/en/flux/installation/configuration/air-gapped.md

```diff
@@ -64,3 +57,33 @@ as the deploy key on your Git server in advance.
 
 For more information on how to use the `flux bootstrap git` command,
 please see the generic Git server [documentation](/flux/installation/bootstrap/generic-git-server/).
+
+## Configure the image pull secret
```

Comment:
Can we name this section something like `Boostrap Flux with a private registry`? This is still about bootrapping, even though there is an extra config strp.

Reply:
In air-gapped the registry is always private.